src/plugins/vis_type_timeseries/common/empty_label.ts

```diff
@@ -11,3 +11,10 @@ import { i18n } from '@kbn/i18n';
 export const emptyLabel = i18n.translate('visTypeTimeseries.emptyTextValue', {
   defaultMessage: '(empty)',
 });
+
+export const getMeaningfulValueOrEmpty = (value: unknown) => {
```

Comment:
Sorry for that but I don't like the word `meaningful` on the function. I think a simple getValueOrEmpty us enough here 🙂

Reply:
np =)